### PR TITLE
Add websites to schema

### PR DESCRIPTION
**Pull request by dylanschultzie**
Simple change that adds network website to chain list. 

If accepted, will go through and add a bunch manually.

Signed-off-by: Dylan Schultz <hello@schultzie.dev>

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Thanks. FYI, there was discussion on this #636 . Under consideration...

**Comment by dylanschultzie** [comment]
Ah looks like you're correct: https://github.com/cosmos/chain-registry/issues/150

**Comment by sunnya97** [comment]
I think I'm personally okay with it 🤷

File: chain.schema.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "$schema": "https://json-schema.org/draft-07/schema",
     "title": "Cosmos Chain",
     "updatelink": "string",
+    "website": "string",
```
Comment (JeremyParish69) [inline chain.schema.json:6]: should I assume that you meant to add 'website' under properties? I don't think you meant to have it up here...

Comment (dylanschultzie) [reply chain.schema.json]: Whoops, thank you. It's been a while since I've messed with json schema tbh.

**Pull request merged**